tests/scripts/build-release.sh

```diff
@@ -18,7 +18,7 @@ function publish() {
     $MAKE -C build/release build BRANCH_NAME=${BRANCH_NAME} TAG_WITH_SUFFIX=${TAG_WITH_SUFFIX} GIT_API_TOKEN=${GIT_API_TOKEN}
     git status &
     git diff &
-    $MAKE -C build/release publish BRANCH_NAME=${BRANCH_NAME} TAG_WITH_SUFFIX=${TAG_WITH_SUFFIX} AWS_ACCESS_KEY_ID=${AWS_USR} AWS_SECRET_ACCESS_KEY=${AWS_PSW} GIT_API_TOKEN=${GIT_API_TOKEN}
+    $MAKE -C build/release publish BRANCH_NAME=${BRANCH_NAME} TAG_WITH_SUFFIX=${TAG_WITH_SUFFIX} GIT_API_TOKEN=${GIT_API_TOKEN} AWS_ACCESS_KEY_ID=${AWS_USR} AWS_SECRET_ACCESS_KEY=${AWS_PSW}
```

Comment:
So having `GIT_API_TOKEN` here with `set -x` above is going to leak these secrets I think. We need to be really careful with this. Please check to see if current GH logs are leaking this info.

If any creds have been leaked, we should rotate these ASAP.

Reply:
I think we are good here ```GIT_API_TOKEN=***```

Reply:
Thanks for looking!